tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/Responses/Response.cs

```diff
@@ -15,26 +15,35 @@ public class Response
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public List<string> ResponseErrors { get; set; }
 
+    [JsonPropertyName("next_steps")]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public string? NextSteps { get; set; }
+
     protected string ToString(StringBuilder value)
     {
         return ToString(value.ToString());
     }
 
     protected string ToString(string value)
     {
-        List<string> errors = [];
+        List<string> messages = [];
```

Comment:
nit: I think "errors" would be clearer here.

Reply:
nextStep isnt necessarily only if there is an error